docs/troubleshooting/Windows/c_watchmaker_logs_salt_call.debug.md

```diff
@@ -0,0 +1,50 @@
+```{eval-rst}
+.. image:: ../../images/cropped-plus3it-logo-cmyk.png
+   :width: 140px
+   :alt: Powered by Plus3 IT Systems
+   :align: right
+   :target: https://www.plus3it.com
+```
+<br>
+
+# The `c:\watchmaker\logs\salt_call.debug` Log-File
+
+This file captures the execution-activities of [SaltStack](https://docs.saltproject.io/en/latest/contents.html) formulae. This file will exist if `watchmaker` has be able to successfully download and install its (SaltStack-based) configuration-content. 
+
+The primary diagnostic interest in this file is if there is an execution-failure within a managed-content module. By default, `watchmaker` will reboot a system after a successful run[^1]. If the expected reboot occurs, this file likely will not be of interest. If the reboot fails to occur and the `watchmaker` log indicates that it was able to start the SaltStack-based operations, then consult this file to identify what failed and (possibly) why.
+
+## Typical Errors
+
+Any errors encountered by SaltStack will typically have a corresponding log-section that starts with a string like:
+
+```
+2023-06-27 12:57:39,841 [salt.state       :325 ][ERROR   ][5656] { ... }
+```
+
+Errors from the failing SaltStack action will typically inclued an embedded JSON-stream. The above snippet's `{ ... }` stands in for an embedded JSON-stream (for brevity's sake). Depending how long the embedded JSON-stream is, it will probably make things easier for the provisionin-user to convert that stream to a more human-readable JSON document-block.
```

Comment:
```suggestion
Errors from the failing SaltStack action will typically include an embedded JSON-stream. The above snippet's `{ ... }` stands in for an embedded JSON-stream (for brevity's sake). Depending how long the embedded JSON-stream is, it will probably make things easier for the provisioning-user to convert that stream to a more human-readable JSON document-block.
```